src/components/Blogcard.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import React from 'react';
+import Box from './Box';
+import Text from './Text';
+
+const Blogcard = ({ imageUrl, url, title }) => {
+	return (
+		<Box mx="mxxxl" target="_blank" cursor="pointer" as="a" href={url}>
+			<Box
+				width={{ mobS: '34.3rem', tabS: '27.6rem', deskM: '38rem' }}
+				height={{ mobS: '20rem', tabS: '16.2rem', deskM: '22rem' }}
+				position="relative"
+				mb="mxl"
+				mt={{ mobS: 'wxs', tabL: 'mxxxl', deskM: 'ws' }}
+			>
+				<Image src={imageUrl} layout="fill" objectFit="cover" />
+			</Box>
+			<Box position={{ mobS: 'relative', tabL: 'absolute' }}>
+				<Text as="h6" color="grey-100" mt="mxxs" width={{ mobS: '34.3rem', tabL: '27.6rem', deskM: '35.4rem' }}>
```

Comment:
use either margin top here or margin bottom in the above element

Reply:
ok